portfolio/src/main/webapp/script/comment-script.js

```diff
@@ -146,27 +147,34 @@ function createImageDiv(imageUrl) {
  * Creates and returns a <div> element containing {@code text}, {@code email},
  * and {@code timeStamp} from comment.
  */
-function createCommentDiv(text, email, timeStamp) {
+function createCommentDiv(text, email, timeStamp, sentimentScore) {
   const commentOuterDiv = document.createElement('div');
   const commentDiv = document.createElement('div');
 
   const textElement = document.createElement('p');
   const emailElement = document.createElement('h4');
+  const scoreElement = document.createElement('h5');
   const dateElement = document.createElement('h5');
 
+  // The comment is most likely good if score is postivie. Otherwise it is most likely bad.
+  var scoreWord = sentimentScore >= 0 ? 'good' : 'bad';
```

Comment:
Please make a const for the number 0. The const name can be "POSITIVE_SENTIMENT_THRESHOLD"

Reply:
fixed